pom.xml

```diff
@@ -695,6 +695,7 @@
       <properties>
         <!-- default config values -->
         <docletName>java-docfx-doclet-1.5.0</docletName>
```

Comment:
Latest release for doclet is v1.6.0 not sure why this wasn't ever updated

Reply:
Let's update it in a separate PR.

Reply:
Sure, I'll create a PR